app/serializers/app.js

```diff
@@ -1,6 +1,15 @@
 import ApplicationSerializer from './application';
 
 export default ApplicationSerializer.extend({
+  normalize() {
```

Comment:
Yes, but the issue is a more fundamental one with the way Ember Data handles `links`.  

The version of ED that we are using has a couple of issues:

* It doesn't lookup the `stack` in `links` it looks for it right in the root of the payload (which doesn't work for us)
* It does not keep track of the `links.self` value of the `Stack` so even if ED uses the `links.stack` value (instead of us having to transpose it into the root of the payload) it would not find the record (since it uses `store.recordForId('stack', "/accounts/my-stack-1")` but the record would only be found with `store.recordForId('stack', "my-stack-1")`).